src/main/java/com/juniorstart/juniorstart/model/User.java

```diff
@@ -3,24 +3,34 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import jakarta.validation.constraints.Email;
 import jakarta.validation.constraints.NotNull;
-import lombok.Getter;
-import lombok.Setter;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.NaturalId;
 
 import javax.persistence.*;
+import java.util.UUID;
 
-
-@Getter
-@Setter
 @Entity
 @Table(name = "users", uniqueConstraints = {
         @UniqueConstraint(columnNames = "email")
 })
-public class User {
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class User  {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
+    @JsonIgnore
+    private UUID privateId;
+
+    @NaturalId
+    private Long publicId;
 
-   
     private String name;
```

Comment:
Rozumiem że name to tak jakby login. Czy pozwalamy na to, żeby mogły być osoby o tym samym nicku?
Też myślę że login musi mieć min. ilość znaków i @ NotBlank. Tak samo Password nie może być empty i musi być minimalna ilość znaków

Reply:
Name to jest imię i nazwisko. Co do hasła to masz rację.

Reply:
Oki nie wiedziałem od czego jest to pole.